src/Resources/views/CRUD/Association/edit_many_script.html.twig

```diff
@@ -520,12 +526,19 @@ This code manages the many-to-[one|many] association field popup
             jQuery('#field_widget_{{ id }}').html("<span><img src=\"{{ asset('bundles/sonataadmin/ajax-loader.gif') }}\" style=\"vertical-align: middle; margin-right: 10px\"/>{{ 'loading_information'|trans([], 'SonataAdminBundle') }}</span>");
             jQuery.ajax({
                 type: 'GET',
+                {# NEXT_MAJOR: Remove the 'code' parameter #}
                 url: '{{ path('sonata_admin_short_object_information', {
-                    'objectId': 'OBJECT_ID',
-                    'uniqid': associationadmin.uniqid,
-                    'code': associationadmin.code,
-                    'linkParameters': sonata_admin.field_description.option('link_parameters', {})
-                })}}'.replace('OBJECT_ID', objectId),
+                        '_sonata_admin': associationadmin.baseCodeRoute,
```

Comment:
this indentation change doesnt look consistent with the other files

Reply:
is it better ?